tools/make-version-header.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Doesn't this break on at least one BSD? (@avsm; @hannesm?)

Reply:
Many BSDs do not install bash by default, period, so there's no bash to be found.

Maybe this has proven to be more trouble than it is worth...